rdkit/Chem/UnitTestSuppliers.py

```diff
@@ -85,17 +85,19 @@ def test3SmilesSupplier(self):
 """
     RDLogger.DisableLog('rdApp.error')
 
-    fileN = tempfile.mktemp('.csv')
+    fileH, fileN = tempfile.mkstemp(suffix='.csv')
     try:
-      with open(fileN, 'w+') as f:
+      with os.fdopen(fileH, 'w') as f:
         f.write(txt)
+        f.close()
```

Comment:
Hi Peter, this is interesting. On a clean checkout of the rdkit repository, I am unable to reproduce the issue I was seeing yesterday. I can confirm that a couple of days ago I could get the test to pass only upon destruction of the supplier, but I cannot reproduce it now, so I'll close this PR.